src/zzz_od/operation/compendium/area_patrol.py

```
@@ -87,16 +87,16 @@ def choose_mission(self) -> OperationRoundResult:
     @node_from(from_name='等待入口加载')
-    @operation_node(name='处理家政券')
-    def handle_coupon(self) -> OperationRoundResult:
-        op = Coupon(self.ctx, self.plan)
-        if self.config.use_coupon:
-            return self.round_by_op_result(op.execute())
-        else:
-            return self.round_success(Coupon.STATUS_CONTINUE_RUN_WITH_CHARGE)
-
-    @node_from(from_name='处理家政券', success=False)
-    @node_from(from_name='处理家政券', status=Coupon.STATUS_CONTINUE_RUN_WITH_CHARGE)
+    # 2.5版本已移除家政券功能，暂时跳过该操作节点
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)